config.go

```diff
@@ -18,32 +18,28 @@ import (
 // Config complete configuration needed to operate the Client.
 type Config struct {
 	// Name to be used in state channels.
-	Alias string
-	// Address will be unlocked in the KeyStore with password
-	KeyStorePath, Password string
-	Address                *Address // OnChain address also calles PerunID.
-	DatabasePath           string   // Path to the database file.
-	ETHNodeURL             string   // URL of the ETH node. Example: ws://127.0.0.1:8545
-	IP                     string   // Ip to listen on.
-	Port                   uint16   // Port to listen on.
+	Alias   string
+	Address *Address // OnChain address also calles PerunID.
+	// On-chain addresses of the Adjudicator and Asset Contract.
+	// In case any of them is nil, the Client will deploy the contract in its
+	// NewClient constructor.
+	Adjudicator, Asset *Address
```

Comment:
```suggestion
	Adjudicator, AssetHolder *Address
```
might otherwise be confused with an ERC20 address for the _asset_.